src/samples/coap/lwm2m-client.c

```diff
@@ -364,7 +350,7 @@ write_security_server_res(void *instance_data, void *user_data,
     switch (res->id) {
     case SECURITY_SERVER_SERVER_URI_RES_ID:
         // TODO: Should I check if exists and free() first?
```

Comment:
I mean I'm a little lost with the memory management. :-(
I'm trying to solve this out, but I believe it will be more clear to me after we talk on monday.